jsonrpc/eth_endpoint.go

```diff
@@ -68,11 +68,17 @@ type ethBlockchainStore interface {
 	GetSyncProgression() *progress.Progression
 }
 
+type ethFilter interface {
+	// FilterExtra filters data from block extra that is not included in block hash
+	FilterExtra(extra []byte) ([]byte, error)
```

Comment:
Is it feasible to move this function to the `ethBlockchainStore` interface?

Reply:
`ethBlockchainStore` interface is implemented by `blockchain.go`. If I am to move this function there, we would have a circular reference import, since `GetIBFTExtraClean` is in `polybft` package, which imports `blockchain` :cry: .